specification/sources/chapters/extensions/oppo/oppo_controller_interaction.adoc

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2023 OPPO
+//
+//
+
+
+include::../meta/XR_OPPO_controller_interaction.adoc[]
+
+*Contributors*::
+    Haomiao Jiang, OPPO +
+    Buyi Xu, OPPO +
+    Yebao Cai, OPPO +
+
+*Overview*
+
+This extension defines a new interaction profile for the OPPO Controller, including but not limited to OPPO MR Glasses Controller.
+
+*OPPO Controller interaction profile*
+
+Interaction profile path:
+* pathname:/interaction_profiles/oppo/oppo_mr_controller
+
+Valid for user paths:
+
+* pathname:/user/hand/left
+* pathname:/user/hand/right
+
+This interaction profile represents the input sources and haptics on the OPPO Controller.
+
+Supported component paths:
+* On /user/hand/left only:
+** subpathname:/input/x/click
+** subpathname:/input/x/touch
+** subpathname:/input/y/click
+** subpathname:/input/y/touch
+** subpathname:/input/menu/click
+** subpathname:/input/heartrate_oppo/value
+
+* On /user/hand/right only:
+** subpathname:/input/a/click
+** subpathname:/input/a/touch
+** subpathname:/input/b/click
+** subpathname:/input/b/touch
+** subpathname:/input/home/click
```

Comment:
You have this marked as system in the XML, which would imply that here you mean:

```suggestion
** subpathname:/input/home/click (may: not be available for application use)
```

If that's not what you mean, then remove the `system` attribute from the XML of this one.